interface/CH.json

```diff
@@ -4116,10 +4116,10 @@
           {
             "in": "body",
             "name": "body",
-            "description": "The client certificate in base64 string format to add.",
+            "description": "The client certificate to add where it's Id must be the certificate's case-insensitive thumbprint and not contains space character.",
```

Comment:
The client certificate to add where its Id must be the certificate's thumbprint without spaces, dashes, or any other special characters. The thumbprint is case-insensitive.